Shared/Sources/ControlsViewController.swift

```diff
@@ -42,6 +42,35 @@ final class ControlsViewController: RevertCollectionViewController {
   private var dataSource: ControlsDataSource?
 }
 
+extension ControlsViewController {
+
+  @available(iOS 13.0, *)
+  override func collectionView(_ collectionView: UICollectionView,
```

Comment:
I tried running the simulator locally and I was not able to load the context menu when the button is tapped.

Looking at the debugger, it appears that the context configuration was not getting called.

Reply:
you might need to long-press it instead of just tap, looks like this is the default behaviour of context menu

Reply:
My bad 🤦‍♀️, I did not long-press the button to trigger it.
Please, just ignore this feedback.